frontend/src/metabase/home/components/Onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,658 @@
+                    </ul>
+                  </Text>
+
+                  {exampleDashboardId && (
+                    <Box data-testid="dashboard-cta">
+                      <Link
+                        to={`/dashboard/${exampleDashboardId}`}
+                        onClick={() =>
+                          trackChecklistItemCTAClicked("dashboard")
+                        }
+                      >
+                        <Button variant="outline">{t`Edit a sample dashboard`}</Button>
+                      </Link>
+                    </Box>
+                  )}
+                </Stack>
+              </Accordion.Panel>
+            </Accordion.Item>
+          </Box>
+          <Box mb={64}>
+            <Title order={2} mb="lg">{t`Get email updates and alerts`}</Title>
+            <Accordion.Item
+              value="subscription"
+              data-testid="subscription-item"
+            >
+              <Accordion.Control icon={<Icon name="subscription" />}>
+                {t`Get dashboard updates by email`}
+              </Accordion.Control>
+              <Accordion.Panel>
+                <Stack spacing="lg">
+                  <VideoTutorial
+                    id="8zjKBpDTEzI"
+                    ref={iframeRefs["subscription"]}
+                    si="9fgftfrqN67Wm7rB"
+                    title="How to create a dashboard email subscription?"
+                  />
+                  {shouldConfigureCommunicationChannels && (
+                    <Text data-testid="subscription-communication-setup">
+                      {jt`${(
+                        <Link
+                          key="subscription-email"
+                          to="/admin/settings/email/smtp"
+                        >{t`Set up email`}</Link>
+                      )} or ${(
+                        <Link
+                          key="subscription-slack"
+                          to="/admin/settings/notifications"
+                        >{t`Slack`}</Link>
+                      )} first.`}
+                    </Text>
+                  )}
+                  <Text>
+                    {jt`To set up a subscription to a dashboard, click on the ${(
+                      <Icon
+                        key="sharing-icon"
+                        name="share"
+                        className={S.inlineIcon}
+                      />
+                    )} ${(
+                      <i key="sharing">{t`sharing`}</i>
+                    )} icon on the top bar, then click on ${(
+                      <Icon
+                        key="subscription-icon"
+                        name="subscription"
+                        className={S.inlineIcon}
+                      />
+                    )} ${(
+                      <b key="subscriptions">{t`Subscriptions`}</b>
+                    )}. On a sidebar on the right set up a dashboard subscription via email or Slack.`}
```

Comment:
(non-blocking comment about i18n):

If we end up being able to use Markdown, this can get a lot simpler. Here's one way to do it, and not necessarily the best way, but perhaps it's one worth thinking about:

```ts
    <MarkdownString
      components={{
        a: ({ href, children }) =>
          href === "#1" ? (
            <>
              <Icon name="share" className={S.inlineIcon} />
              &nbsp;<i>{children}</i>
            </>
          ) : (
            <>
              <Icon
                name="subscription"
                className={S.inlineIcon}
              />
              &nbsp;<b>{children}</b>
            </>
          ),
      }}
    >
      {t`To set up a subscription to a dashboard, click on the [sharing](#1) icon on the top bar, then click on [Subscriptions](#2). On the right-hand sidebar set up a dashboard subscription via email or Slack.`}
    </MarkdownString>
```

In this case we could also use `*sharing* icon ... click on **Subscriptions**`, but note that with link syntax – notwithstanding its hackiness – we can have any number of distinctly marked-up substrings.